engines/bops_api/app/controllers/concerns/bops_api/param_helpers.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module BopsApi
+  module ParamHelpers
+    extend ActiveSupport::Concern
+
+    # when a parameter is an array of comma-separated values, this method will split them into an array
+    def handle_comma_separated_param(permitted, key)
+      Array(permitted[key]).flat_map { |v| v.to_s.split(",") }.compact_blank.uniq
+    end
```

Comment:
This could probably just take a single parameter, i.e., the individual param, instead of the entire params object and the key into it.

```suggestion
    def handle_comma_separated_param(param)
      Array(param).flat_map { |v| v.to_s.split(",") }.compact_blank.uniq
    end
```